File: Go/Repeat.go
package main

func Repeat() {

	print(ln{1},"これから繰り返し処理を試します",ln{2})

	print(ln{1},"for n:=0;n<3;n++ {…}",ln{1})
	for n:=0;n<3;n++ {
		print("n=",n,"のループ",ln{1})
	}

	print(ln{1},"for n,c := range array {…}",ln{1})
	var cs = []string{"壱","弐","参"}
	for n,c := range cs {
		print(n,"番目 c=\"",c,"\"のループ",ln{1})
		// n,cと2つ並んでいるが,不要な要素があれば,_で置くことで,変数の代わりになる
		// e.g. n,c → _,c
	}

	print(ln{1},"for ~ {…}",ln{1})
	str := ""
	for len(str)<6 {
		str+="*"
		print(str,ln{1})
	}
	/*
		無限ループは for {…} により作れる。break でループから抜ける。
	*/

	print(ln{3})

}

File: Python/Repeat.py
#! /usr/bin/env python3
# -*- coding: utf-8 -*-

from Utility import *

def Repeat():

	pnl(1)

	prints("これから繰り返し処理を試します",nl(2))

	prints("for n in range(0,3):…else:…",nl(2))
	for n in range(0,3):
		print(f"n={n} のループ")
	else:
		print("これでループは終わり")

	pnl(1)

	prints("for c in array:…else:…",nl(2))
	cs=("壱","弐","参")
	for c in cs:
		print(f"c=\"{c}\" のループ")
	else:
		print("これでループは終わり")

	pnl(1)

	prints("while ~:…else:…",nl(2))
	string=""
	while len(string)<6:
		string+="*"
		print(string)
	else:
		print("++++++")

	pnl(2)

runningDirectly(__name__,Repeat)